react-native-libraries.json

```diff
@@ -13695,5 +13695,29 @@
     "android": true,
     "web": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/WrathChaos/react-native-segmented-control-2",
+    "examples": [
+      "https://github.com/WrathChaos/react-native-segmented-control-2/tree/master/example"
+    ],
+    "images": [
+      "https://github.com/WrathChaos/react-native-segmented-control-2/blob/master/assets/Screenshots/react-native-segmented-control-2.gif"
+    ],
+    "alternatives": [
+      "@react-native-segmented-control/segmented-control",
+      "rn-segmented-tab-controls",
+      "react-native-paper",
+      "react-native-segmented-control-tabs"
+    ],
+    "ios": true,
+    "android": true,
+    "web": true,
+    "windows": true,
+    "macos": true,
+    "tvos": true,
+    "visionos": true,
```

Comment:
Adding out-of-tree platforms support requires a dedicated example, or reference to an OSS app using it on that platform:
* https://github.com/react-native-community/directory?tab=readme-ov-file#%EF%B8%8F-out-of-tree-platforms

I can only see the regular example for build-in native platforms in the repository code. To do not block the addition, we can update the entry later in time, when we have some references for out-of-tree usage.

Reply:
ahh ok, i will just leave ios, android, and web then